docs/Allocene/style/main-login.css

```diff
@@ -0,0 +1,207 @@
+@import url("./header.css");
+@import url("./footer.css");
+@import url("./font.css");
+
+main {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: calc(100vh - 160px);
+  min-width: 400px;
+}
+
+.contain {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 70px;
+}
+
+.content {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: 80px;
+  margin-bottom: 150px;
+}
+
+.form_h1 {
+  width: 100%;
+}
+
+.h1 {
+  font-size: 48px;
+  line-height: 32px;
+  font-weight: 700;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.form__login {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.item {
+  display: flex;
+  flex-direction: column;
+  margin-top: 40px;
+}
+
+.item__check {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 40px;
+  margin-right: 12px;
+  margin-left: 0;
+}
+
+.item__button {
+  display: flex;
+  flex-direction: column;
+}
+
+.label {
+  font-size: 22px;
+  font-weight: 600;
+}
+
+.inp {
+  box-sizing: border-box;
+  height: 70px;
+  width: 575px;
+  font-size: 24px;
+  margin-top: 22px;
+  border-radius: 6px;
+  outline: 0;
+  border: solid .6px #A9A9A9;
+  padding-left: 19px;
+}
+
+.inp:focus {
+  border-bottom: solid 6px #EF4934;
+}
+
+.inp::placeholder {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.inp__check {
+  height: 0;
+  width: 0;
+  opacity: 0;
+  position: absolute;
+}
+
+.custom__check {
+  position: relative;
+  display: inline-block;
+  height: 20px;
+  width: 20px;
+  background-color: #FFF;
+  border: solid 1px #EF4934;
+  border-radius: 6px;
+  margin-right: 12px;
+  margin-bottom: 0;
+  vertical-align: sub;
+  z-index: -1;
+}
+
+.custom__check::before {
+  content: " ";
+  display: inline-block;
+  width: 15px;
+  height: 15px;
+  background-image: url("/homeworks/artur.kobyliatsky_Allocene/GamesProject/img/check.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  margin-top: 2px;
+  transition: .2s ease-in;
+}
+
+.inp__check:checked+.custom__check::before {
```

Comment:
This style doesn't work - it is impossible to understand if the checkbox is checked or not 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/b1b9e810-2b8d-4ad3-b864-903b6ea3e70e)

Reply:
Incorrect path to the image, Done